data/human/human missions.txt

```diff
@@ -4077,3 +4077,135 @@ mission "Laborer Payment"
+					goto credits
+				`	"Hold on to the credits. I'm a captain; I can always make more."`
+			action
+				payment -13000
+			`	After a moment of stunned silence, she manages a choked "Thank you." You notice her eyes are brimming with tears.`
+				goto choice1
+			label credits
+			`	"Not really, no. Honestly, this is most of my savings... but... it's my dad."`
+			label choice
+			choice
+				`	"Do you think I could come along? I've heard so much about your father."`
+					goto father
+				`	"Thank you, and good luck."`
+					goto end
+				`	"Hold on to the credits. I'm a captain; I can always make more."`
+			action
+				payment -13000
+			`	After a moment of stunned silence, she manages a choked "Thank you." You notice her eyes are brimming with tears.`
+			label choice1
+			choice
+				`	"Do you think I could come along? I've heard so much about your father."`
+				`	"Thank you, and good luck."`
+					goto end
+			label father
+			`	She is taken aback at first, but agrees to let you join, and you wind your way away from the landing pads, across rural, wooded countryside, and finally to a modest house nestled along a valley edge. Riley knocks, and a nurse lets you in.`
+			`	Her father is in a comfortable-looking medical cradle. Wires flow from it to the holoscreens around the room, showing vitals and other indecipherable information. Despite his wan, strained face and atrophied body, he raises his head to look at his daughter.`
+			`	"Ri," he says, face lighting up. "My love. You cannot know how happy it makes me to see you. And you have company, I see. My name's Church."`
+			choice
+				`	"Riley chartered a flight here aboard my ship. I heard so much about you on the way, it's a pleasure to meet you."`
+				`	Riley and I have gotten to know each other on the long flight here. You've raised an incredible woman."`
+			`	"<first> was the only captain willing to fly here from <origin>. I wouldn't be here without them," she says.`
+			`	He smiles at you, acknowledging your part in their reunion with an appreciative nod.`
+			`	You swap stories for hours, flowing between reminiscing over old times and catching up on their lives over the past few years. You learn that since he was young, Church researched AI that cannot develop into a threat; a project that the Deep has been quietly interested in.`
+			`	Riley, in turn, had always surrounded herself with music, and moved to <origin> to share her passion with other like-minded people. You talk about what it's like to call a ship your home, leaving behind everything you grew up knowing and embracing life as a starship captain.`
+			`	Finally, her father lies back. "Well, thank you for making the trip out to see your old man. I... have to go now, I think. Don't forget about me, hear? And," Church hesitates for a moment, "whatever happens, please don't think ill of me. I will always love you."`
+			`	His body relaxes, and he's gone.`
+			choice
+				`	(Stay comfortingly close.)`
```

Comment:
Having a third option that might leave more (emotional) distance between the two of you would probably be welcome here, especially with how this can end. It could still go to the `close` label, since that is just sitting down, but the two options both have a sense of intimacy for someone you've only know for a week or two.